index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Challenge #1 - Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="theme-color" content="#47D95E">
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <img class="header__img" src="assets/undraw_developer_activity_bv83.svg" alt="An illustration of a developer sitting at a desk with a computer">
+            <h1 class="header__caption">Code Academy</h1>
+            <img class="header__ribbon" src="assets/coming-soon-ribbon.png" alt="Coming soon ribbon">
+        </header>
+        <form action="" method="get" class="emailForm">
+            <label for="emailForm__input" class="emailForm__title">Leave your email</label>
+            <input type="email" name="emailForm__input" id="emailForm__input" class="emailForm__input" placeholder="E-mail" required>
```

Comment:
W name raczej nie stosuje się Bema, z reguły są unikalne więc nie ma to sensu.

Reply:
W CSSie też nie stosował bym camelcase
email-form wygląda o wiele lepiej a jeszcze lepiej form-email bo od razu wiadomo jaki element HTML jest blockiem